package.json

```diff
@@ -49,6 +49,7 @@
     "eslint-plugin-jsx-a11y": "^1.5.3",
     "eslint-plugin-react": "^5.2.2",
     "jsdom": "9.4.1",
+    "medium-editor": "^5.22.1",
```

Comment:
This isn't needed anymore. Some dude coded up a PR where we are now using the trumbo WYSIWYG editor ;)